### DISCUSSION TITLE
Beginner mode

### DISCUSSION THREAD

**Discussion by welon68** [iNavFlight/inav#2167] [Ideas]
Hello. I am very interested in the project "Inav" but I'm a weak pilot. Is it possible to design a "**Beginner mode**"? I imagine it as flying in a virtual cylinder, a cylinder of a certain diameter or radius and a corresponding height. These parameters were to be set in the program INAV. Copter would not run away :-)
Best wishes 
welon68

**Comment by digitalentity** [comment]
Duplicate of #62
